package.json

```diff
@@ -9,8 +9,9 @@
     "bundle": "wireit",
     "lint": "wireit",
     "lint:fix": "wireit",
-    "lint:cpp": "find src packages/realm/bindgen/src -name '*.cpp' -or -name '*.h' -or -name '*.hpp' | xargs clang-format --dry-run --Werror",
-    "lint:cpp:fix": "find src packages/realm/bindgen/src -name '*.cpp' -or -name '*.h' -or -name '*.hpp' | xargs clang-format -i",
+    "trunk:check": "trunk check",
+    "trunk:check-all": "trunk check --all",
+    "trunk:fmt": "trunk fmt",
```

Comment:
Any reason for not calling it `lint:cpp:fix`?

Reply:
That would encode an implicit expectation that the `.trunk/trunk.yaml` only enables the `clang-format` linter, introducing a coupling between this name and the configuration file, which I'd like to avoid.

Reply:
See the answer above. I've decided to call trunk in the existing `lint` and `lint:fix` instead.